scripts/build.ps1

```diff
@@ -28,7 +28,9 @@ Param(
     [string]$LogLib = 'zlog',
     # Log folder to use for DU logs
     # TODO(JeffMill): Change this when folder structure determined.
-    [string]$LogDir = '/var/log/adu'
+    [string]$LogDir = '/var/log/adu',
+    # Running in pipeline?
+    [switch]$Pipeline
```

Comment:
i'd prefer "-NoPrompt" (currently i trigger off of "-Clean") or perhaps checking if we're running in an interactive console or not

[Environment]::UserInteractive maybe?